### ISSUE TITLE
S3 bug

### ISSUE THREAD

**Issue by stack72** [dtan4/terraforming#116]
I am attempting to try and collect all the S3 buckets I have in my infrastructure:

```
terraforming s3
/opt/rubies/2.0.0-p451/lib/ruby/gems/2.0.0/gems/aws-sdk-core-2.1.3/lib/seahorse/client/plugins/raise_response_errors.rb:15:in `call': The bucket you are attempting to access must be addressed using the specified endpoint. Please send all future requests to this endpoint. (Aws::S3::Errors::PermanentRedirect)
	from /opt/rubies/2.0.0-p451/lib/ruby/gems/2.0.0/gems/aws-sdk-core-2.1.3/lib/aws-sdk-core/plugins/s3_sse_cpk.rb:18:in `call'
	from /opt/rubies/2.0.0-p451/lib/ruby/gems/2.0.0/gems/aws-sdk-core-2.1.3/lib/aws-sdk-core/plugins/param_converter.rb:21:in `call'
	from /opt/rubies/2.0.0-p451/lib/ruby/gems/2.0.0/gems/aws-sdk-core-2.1.3/lib/seahorse/client/plugins/response_target.rb:18:in `call'
	from /opt/rubies/2.0.0-p451/lib/ruby/gems/2.0.0/gems/aws-sdk-core-2.1.3/lib/seahorse/client/request.rb:70:in `send_request'
	from /opt/rubies/2.0.0-p451/lib/ruby/gems/2.0.0/gems/aws-sdk-core-2.1.3/lib/seahorse/client/base.rb:207:in `block (2 levels) in define_operation_methods'
	from /opt/rubies/2.0.0-p451/lib/ruby/gems/2.0.0/gems/terraforming-0.1.6/lib/terraforming/resource/s3.rb:46:in `bucket_policy_of'
	from (erb):5:in `block in apply_template'
	from (erb):1:in `each'
	from (erb):1:in `apply_template'
	from /opt/rubies/2.0.0-p451/lib/ruby/2.0.0/erb.rb:849:in `eval'
	from /opt/rubies/2.0.0-p451/lib/ruby/2.0.0/erb.rb:849:in `result'
	from /opt/rubies/2.0.0-p451/lib/ruby/gems/2.0.0/gems/terraforming-0.1.6/lib/terraforming/util.rb:4:in `apply_template'
	from /opt/rubies/2.0.0-p451/lib/ruby/gems/2.0.0/gems/terraforming-0.1.6/lib/terraforming/resource/s3.rb:19:in `tf'
	from /opt/rubies/2.0.0-p451/lib/ruby/gems/2.0.0/gems/terraforming-0.1.6/lib/terraforming/resource/s3.rb:7:in `tf'
	from /opt/rubies/2.0.0-p451/lib/ruby/gems/2.0.0/gems/terraforming-0.1.6/lib/terraforming/cli.rb:134:in `tf'
	from /opt/rubies/2.0.0-p451/lib/ruby/gems/2.0.0/gems/terraforming-0.1.6/lib/terraforming/cli.rb:129:in `execute'
	from /opt/rubies/2.0.0-p451/lib/ruby/gems/2.0.0/gems/terraforming-0.1.6/lib/terraforming/cli.rb:108:in `s3'
	from /opt/rubies/2.0.0-p451/lib/ruby/gems/2.0.0/gems/thor-0.19.1/lib/thor/command.rb:27:in `run'
	from /opt/rubies/2.0.0-p451/lib/ruby/gems/2.0.0/gems/thor-0.19.1/lib/thor/invocation.rb:126:in `invoke_command'
	from /opt/rubies/2.0.0-p451/lib/ruby/gems/2.0.0/gems/thor-0.19.1/lib/thor.rb:359:in `dispatch'
	from /opt/rubies/2.0.0-p451/lib/ruby/gems/2.0.0/gems/thor-0.19.1/lib/thor/base.rb:440:in `start'
	from /opt/rubies/2.0.0-p451/lib/ruby/gems/2.0.0/gems/terraforming-0.1.6/bin/terraforming:5:in `<top (required)>'
	from /opt/boxen/rbenv/versions/2.0.0-p451/bin/terraforming:23:in `load'
	from /opt/boxen/rbenv/versions/2.0.0-p451/bin/terraforming:23:in `<main>'
```

I think it is trying to collect us-standard buckets even though my credentials point to eu-west-1

Is there something I am missing here?

Paul

**Comment by dtan4** [comment]
I fixed this bug at #121 to export bucket in the same region as the region you set.
This fix was released at v0.2.0 :shipit: 

If you still have something wrong, please reopen this issue.

**Comment by stack72** [comment]
@dtan4 thanks for fixing this - works perfectly :)
